### ISSUE TITLE
Investigate squashfs under secureboot

### ISSUE THREAD

**Issue by Itxaka** [rancher/elemental#487]

**Comment by kkaempf** [comment]
We asked for squashfs in grub but retracted this request later :-/
(The fewer modules the better - from a security pov. So security and grub2 maintainer were actually happy about this :wink:)

**Comment by Itxaka** [comment]
Yes and this is good. We only need to support loop devices in Teal so we went the route of petitioning for that to the grub maintainers.

This is just a follow up card to understand if we need it, or not or we can live with it, etc... Also following the discussion on dropping squashfs from https://github.com/rancher/elemental-toolkit/discussions/1626 which would result in this closing. Mostly a side effect to show that there is things we miss if we support squashfs.
